docs/site/downloads/cldr-45.md

```diff
@@ -0,0 +1,143 @@
+2.  The new SI prefixes are provided for metric units so that implementations can verify that they are providing the correct prefix values, including the new prefixes (see [New Prefixes in the SI: ronto, quecto, ronna, quetta](https://blog.ansi.org/anab/new-prefixes-si-ronto-quecto-ronna-quetta/)).
+3.  In last release CLDR 44, Unicode updated its outbound license from the "[Unicode, Inc. License - Data Files and Software](https://opensource.org/license/unicode-inc-license-agreement-data-files-and-software)" to the "[Unicode License v3](https://opensource.org/license/unicode-license-v3)". All of the substantive terms of the license remain the same. The only changes made were non-substantive technical edits. The new license is OSI-approved and has been assigned the SPDX Identifier Unicode-3.0.
+4.  Keyboard has a new DTD (keyboard3.dtd and the `<keyboard3>` element). There have been some incompatible changes between the tech preview and the stable version now in v44.
+5.  Redundant values that inherit “laterally” may be removed in production data: Some data values inherit “sideways” from another element with the same parent, in the same locale. This has been long specified in [LDML: Lateral\_Inheritance](https://www.unicode.org/reports/tr35/#Lateral_Inheritance), but some clients may not implement to the specification, and get the wrong answers. For example, consider the following items in the en locale, some added in CLDR 44 to provide clients a way to explicitly select a particular variant across locales (instead of using the default):
+```
+<territory type="IO">British Indian Ocean Territory</territory> <!-- The locale default, matches one of the alt forms -->  
+<territory type="IO" alt="biot">British Indian Ocean Territory</territory> <!-- explicit "biot" variant" -->  
+<territory type="IO" alt="chagos">Chagos Archipelago</territory> <!-- explicit "chagos" variant" --> 
+```
+    Both `alt` forms inherit sideways from the non-`alt` form. Thus in this case, the "`biot`" variant is redundant and will be removed in production data. Clients that are trying to select the "`biot`" variant but find it missing should fall back to the non-`alt` form. Similar behavior occurs with plural forms for units, where some plural forms may match and thus fall back to the "`other`" form.
+
+## [Known Issues](https://unicode-org.atlassian.net/issues/CLDR-17535?jql=project%20%3D%20cldr%20and%20labels%20%3D%20%22ReleaseKnownIssue%22%20and%20status%20!%3D%20done)
+
+-   [CLDR-17095](https://unicode-org.atlassian.net/browse/CLDR-17095). The region-based firstDay value (see [weekData](https://www.unicode.org/reports/tr35/tr35-72/tr35-dates.html#Week_Data)) is currently used for several different purposes. In the future, some of these functions will be separated out:
```

Comment:
we could probably linkify CLDR- keywords.

Reply:
That would be handy.